module/applications/actor/short-rest.mjs

```diff
@@ -36,26 +36,29 @@ export default class ShortRestDialog extends Dialog {
 
   /** @inheritDoc */
   getData() {
-    const data = super.getData();
+    const context = super.getData();
+    context.isGroup = this.actor.type === "group";
```

Comment:
hi sorry to butt in.
im working on a module subtype that is an extension of the group actor. Could these properties be moved to a set in the application's class such that a module could determine which actors are valid rather than hardcoding it to only `"group"`?

Reply:
I suppose, though the only place this is used is to determine a variation of the resting message, nothing functionality-wise is dependent on it.